notebooks/official/bigquery-ml/bqml-vertexai-model-registry.ipynb

```diff
@@ -0,0 +1,1465 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    ! bq rm -r -f $PROJECT_ID:$BQ_DATASET_NAME
</pre><p>You can use the BQ client from above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>Do you think using that makes more sense or this is fine too? </p>

Reply:
That is also fine as it's quite common.

Reply:
Ok I will keep it then!